### PR TITLE
New branch with GSRS 3.1 import architecture

### PR DESCRIPTION
**Pull request by ChemMitch**
to be tested by brave people.

### CODE REVIEW

File: gsrs-fda-substance-extension/src/main/java/fda/gsrs/substance/exporters/FDACodeExporter.java [hunk 1]
```diff
@@ -1,83 +1,57 @@
 package fda.gsrs.substance.exporters;
 
-import com.fasterxml.jackson.databind.JsonNode;
```
Comment (tylerperyea) [inline gsrs-fda-substance-extension/src/main/java/fda/gsrs/substance/exporters/FDACodeExporter.java:3]: Need to revert this one to master. Looks like it got lost in the shuffle.

File: gsrs-fda-substance-extension/src/main/java/fda/gsrs/substance/exporters/FDANameExporter.java [hunk 2]
```diff
@@ -1,178 +1,143 @@
 package fda.gsrs.substance.exporters;
-import com.fasterxml.jackson.databind.JsonNode;
+
```
Comment (tylerperyea) [inline gsrs-fda-substance-extension/src/main/java/fda/gsrs/substance/exporters/FDANameExporter.java:2]: Need to revert this one to master. Looks like it got lost in the shuffle.

File: gsrs-fda-substance-extension/src/main/java/fda/gsrs/substance/exporters/FDARelationshipExporter.java [hunk 3]
```diff
@@ -1,14 +1,13 @@
 package fda.gsrs.substance.exporters;
 
-import com.fasterxml.jackson.databind.JsonNode;
```
Comment (tylerperyea) [inline gsrs-fda-substance-extension/src/main/java/fda/gsrs/substance/exporters/FDARelationshipExporter.java:3]: Need to revert this one to master. Looks like it got lost in the shuffle.

File: gsrs-module-substance-example/src/test/java/example/substance/tasks/DEAScheduledTaskInitializerTest.java [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+            factory.addValidator("substances", config);
+        }
+
+        File dataFile = new ClassPathResource(fileName).getFile();
+        loadGsrsFile(dataFile);
+    }
+
+    @Test
+    public void DeaScheduledTaskTest1() throws IOException {
+        String reportFilePath ="d:\\temp\\task_test.report.txt";
```
Comment (tylerperyea) [inline gsrs-module-substance-example/src/test/java/example/substance/tasks/DEAScheduledTaskInitializerTest.java:57]: This needs to be fixed. Can't use D:\ in tests, must use java temporary file path

File: gsrs-module-substance-example/src/test/resources/application-test.conf [hunk 5]
```diff
@@ -4,39 +4,34 @@ spring.main.allow-bean-definition-overriding=true
...
+logging.level.gsrs.controller=trace
+logging.level.example.imports.readers=trace
+logging.level.gsrs.dataexchange.processing_actions=trace
+
+
+logging.level.gsrs=TRACE
 logging.level.example.substance=ERROR
 logging.level.ix.ginas=ERROR
 logging.level.org.springframework=ERROR
+logging.level.example.imports=trace
```
Comment (tylerperyea) [inline gsrs-module-substance-example/src/test/resources/application-test.conf:32]: A lot of these trace logs may need to change for jenkins. Too verbose a log makes the build fail due to the log getting too big. Can we comment out these?

**Review by tylerperyea — CHANGES_REQUESTED** [review]
A few changes got clobbered with exports that need to be cherry picked back from master, and there are a couple of other minor things to fix, but this looks good!

File: gsrs-module-substance-example/src/main/resources/application-postgres.conf [hunk 6]
```diff
@@ -1,6 +0,0 @@
-include "application.conf"
```
[the remark this answers is not in the document]
Comment (ChemMitch) [reply gsrs-module-substance-example/src/main/resources/application-postgres.conf:1]: It had my personal DB info.
I can come up with a generic config file for pg

File: gsrs-module-substance-example/src/test/java/example/substance/tasks/DEAScheduledTaskInitializerTest.java (hunk 4 above)
Comment (ChemMitch) [reply gsrs-module-substance-example/src/test/java/example/substance/tasks/DEAScheduledTaskInitializerTest.java:57]: Darn.  I had fixed these earlier. Will do again.

**Review by tylerperyea — APPROVED** [review]

**Pull request merged**